_config.yml

```diff
@@ -30,6 +30,11 @@ disqus_shortname:                                       # Your discus shortname
 cusdis_app_id:                                          # Your cusdis data-app-id
 color_theme: auto                                       # auto, dark or light
 
+# Social icons
+me:
+  github: "sylhare"
+  stack-overflow: "7747942/sylhare"
```

Comment:
Why is there social icons in the `_config.yaml` and the `_data/icons.yml`?

Let's have like before a `social.yaml` where you put your username (github, stackoverflow, ...) and the `icons.yaml` could be just the boilerplate to actually build the icons